docs/customization/README.md

```diff
@@ -0,0 +1,3 @@
+* [Diffing](diff.md)
+* [Synchronization](synchronization.md)
+* [Health Checks](health.md)
```

Comment:
I don't think we need a nested level of docs for "customization". The documentation for customizing health can go under the existing health.md. It's more natural to have the docs for built-in health checks and custom health checks to be part of the same document.

Reply:
Agree. Fixed